### PR TITLE
update ATO checklist

### PR DESCRIPTION
**Pull request by afeld**
Part of https://github.com/18F/tts-tech-portfolio/issues/1359. Paired on this with @rpalmer-gsa, so should be good to go.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ato.md [hunk 1]
```diff
@@ -5,21 +5,24 @@ title: ATO for [system] - due [date]
...
 
 ## TODOs
 
 If your system isn't live yet, "production" refers to the environment that _will_ be production.
 
 ### Phase 0: As early in the project as possible
 
+#### Project team
 
+- [ ] Request ISSO
```
Comment (adborden) [inline .github/ISSUE_TEMPLATE/ato.md:25]: How do I do that?

File: .github/ISSUE_TEMPLATE/ato.md [hunk 2]
```diff
@@ -82,35 +93,99 @@ These tasks apply to every repository/application/hostname/language that is dire
...
+The following penetration tests will be performed:
+
+- External Interfaces
+- OWASPv4.1 & PTES-TG
+- Gray Box
+- Authenticated
+
+#### Project team
+
+- [ ] Environment being tested is frozen
```
Comment (adborden) [inline .github/ISSUE_TEMPLATE/ato.md:160]:
```suggestion
- [ ] Environment being tested is frozen (disable automated deploys to production)
```

**Review by adborden — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/ato.md (hunk 1 above)
Comment (afeld) [reply .github/ISSUE_TEMPLATE/ato.md:25]: @rpalmer-gsa How's this?

```suggestion
- [ ] Request ISSO from the ISSM
```

**Comment by JJediny** [comment]
Terraform plan in __terraform__
<details>
  <summary>No changes. Infrastructure is up-to-date.</summary>

```
No changes. Infrastructure is up-to-date.

This means that Terraform did not detect any differences between your
configuration and real physical resources that exist. As a result, no
actions need to be performed.
```
</details>

Plan generated in [plan #6](https://github.com/18F/tts-tech-portfolio/actions/runs/969697046)

**Review by JJediny — APPROVED** [review]
Looks great

File: .github/ISSUE_TEMPLATE/ato.md (hunk 1 above)
Comment (rpalmer-gsa) [reply .github/ISSUE_TEMPLATE/ato.md:25]: Yep that looks good. We are working on a intake form but nothing ready yet.

**Pull request merged**